SokolTextGame/Commands.cs

```diff
@@ -46,15 +47,67 @@ public void Execute(string[] words, World world)
         {
             if (string.Join(" ", words) == "where") Console.Write("Where? Where what? Or where who?\n");
             else if (string.Join(" ", words) == "where am i") Console.Write(world.CurrentLocation().Description);
-            else Console.Write("I know the command \"where am i\"\n");
+            else if (string.Join(" ", words) == "where can i go")
+            {
+                string output = string.Join("", world.CurrentLocation().possibleLocation);
+                Console.ForegroundColor = ConsoleColor.Green;
+                Console.WriteLine($"Possible location : {output}");
+                Console.ResetColor();
+            }
+            else Console.Write("I know the command \"where am i\" and \"where can i go\"\n");
+        }
+    }
+    public class GoTo : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words.Length == 3)
+            {
+                if (string.Join(" ", words.Take(2)) == "go to")
```

Comment:
Generally, using two nested ifs one after another is considered bad code. Please collapse them into a single if and use a logical and (&&) to check if they're both true at the same time.
Also, I would advise to change the `words.Length == 3` to `words.Length > 2` in case one of your future locations has a multi-word name. Example: `weapon shop` which you're going to be implementing next.